tests/test_iac_controller.py

```diff
@@ -0,0 +1,44 @@
+import responses
```

Comment:
I might suggest using something similar to the following to ensure that the count mechanism of the controller is working as expected:

```python
import json

import responses

from iac_controller import IACController
from tests import IACConfigTestCase


class TestIACController(IACConfigTestCase):
    iac_batches = {'0': [], '1': ['testiaccode3'], '2': ['testiaccode1', 'testiaccode2']}

    @staticmethod
    def request_callback(request):
        payload = json.loads(request.body)
        resp_body = TestIACController.iac_batches[payload['count']]
        return (200, {}, json.dumps(resp_body))

    def test_get_iac(self):
        # Given
        with responses.RequestsMock(assert_all_requests_are_fired=True) as rsps:
            rsps.add_callback(
                responses.POST, f'{self.test_iac_url}/iacs',
                callback=self.request_callback,
                content_type='application/json'
            )
            iac_controller = IACController(max_total_iacs=1, batch_size=1)

            # When
            iac = iac_controller.get_iac()

        # Then
        assert iac == self.iac_batches['1'][0]

    def test_get_iac_multiple_batches(self):

        # Given
        with responses.RequestsMock(assert_all_requests_are_fired=True) as rsps:
            rsps.add_callback(
                responses.POST, f'{self.test_iac_url}/iacs',
                callback=self.request_callback,
                content_type='application/json'
            )
            iac_controller = IACController(max_total_iacs=3, batch_size=2)

            # When
            first_iac = iac_controller.get_iac()

            second_iac = iac_controller.get_iac()

            third_iac = iac_controller.get_iac()

        # Then
        assert first_iac in self.iac_batches['2']
        assert second_iac in self.iac_batches['2']
        assert third_iac == self.iac_batches['1'][0]
```